### ISSUE TITLE
Airflow lineage: AttributeError: 'AirflowConfigParser' object has no attribute 'getjson'

### ISSUE THREAD

**Issue by TheZbunj** [open-metadata/OpenMetadata#5885]
**Affected module**
Ingestion Framework

**Describe the bug**
When running openmetadata/airflow:0.11.0 docker image and setting lineage environment variables to use azure auth provider, I'm getting an error on container startup:

```
Traceback (most recent call last):
  File "/home/airflow/.local/bin/airflow", line 8, in <module>
    sys.exit(main())
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/__main__.py", line 40, in main
    args.func(args)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/cli/cli_parser.py", line 47, in command
    func = import_string(import_path)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/utils/module_loading.py", line 32, in import_string
    module = import_module(module_path)
  File "/usr/local/lib/python3.9/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/cli/commands/db_command.py", line 24, in <module>
    from airflow.utils import cli as cli_utils, db
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/utils/db.py", line 26, in <module>
    from airflow.jobs.base_job import BaseJob  # noqa: F401
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/jobs/__init__.py", line 19, in <module>
    import airflow.jobs.backfill_job
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/jobs/backfill_job.py", line 29, in <module>
    from airflow import models
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/__init__.py", line 20, in <module>
    from airflow.models.baseoperator import BaseOperator, BaseOperatorLink
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/baseoperator.py", line 196, in <module>
    class BaseOperator(Operator, LoggingMixin, TaskMixin, metaclass=BaseOperatorMeta):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/baseoperator.py", line 941, in BaseOperator
    def post_execute(self, context: Any, result: Any = None):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/lineage/__init__.py", line 103, in apply_lineage
    _backend = get_backend()
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/lineage/__init__.py", line 61, in get_backend
    return clazz()
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow_provider_openmetadata/lineage/openmetadata.py", line 58, in __init__
    _ = get_lineage_config()
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow_provider_openmetadata/lineage/config/loader.py", line 80, in get_lineage_config
    return parse_airflow_config(airflow_service_name)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow_provider_openmetadata/lineage/config/loader.py", line 56, in parse_airflow_config
    security_config = load_security_config_fn()
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow_provider_openmetadata/lineage/config/providers.py", line 100, in load_azure_auth
    scopes=conf.getjson(LINEAGE, "scopes", fallback=[]),
AttributeError: 'AirflowConfigParser' object has no attribute 'getjson'
```

**To Reproduce**

Run openmetadata/airflow:0.11.0 docker image and set lineage environment variables:
```
AIRFLOW__LINEAGE__AIRFLOW_SERVICE_NAME: "airflow"
AIRFLOW__LINEAGE__AUTHORITY: <authority>
AIRFLOW__LINEAGE__AUTH_PROVIDER_TYPE: "azure"
AIRFLOW__LINEAGE__BACKEND: "airflow_provider_openmetadata.lineage.openmetadata.OpenMetadataLineageBackend"
AIRFLOW__LINEAGE__CLIENT_ID: <id>
AIRFLOW__LINEAGE__CLIENT_SECRET: <secret>
AIRFLOW__LINEAGE__OPENMETADATA_API_ENDPOINT: <endpoint>
AIRFLOW__LINEAGE__SCOPES: [<scopes>]
```

**Expected behavior**
Container parses scopes configuration and starts airflow.

**Version:**
 - OS: 
 - Python version:
 - OpenMetadata version: 0.11.0
 - OpenMetadata Ingestion package version: 0.11.0

**Additional context**
Seems to be related to #5507

**Comment by pmbrull** [comment]
hi @TheZbunj, which Airflow version are you using?

**Comment by pmbrull** [comment]
oki, digging a bit into this, the `getjson` property is available in `main` but not for 2.1.4:
- [main](https://github.com/apache/airflow/blob/main/airflow/configuration.py#L683)
It was not introduced until [2.3](https://github.com/apache/airflow/blob/v2-3-stable/airflow/configuration.py). We'll take care of this. Thanks
